src/components/Header.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react'
+import { view , store } from 'react-easy-state';
+import Badge from './Badge'
+import SearchBar from './SearchBar'
+import { BrowserRouter as Router, Redirect, Link } from "react-router-dom"
+
+interface HeaderProps {
+    title:any,
+    package?:boolean,
+    version?:any,
+    grade?:any,
+}
+
+class Header extends React.Component<HeaderProps>{
+    render(){
```

Comment:
Seeing as you're not using the "classness" of this component, it may be better to change it to the simpler `view(() => {
    return <>
})`